src/frontend/scss/generic/_accessibility.scss

```diff
@@ -2,23 +2,19 @@
 // to users that need it.
 
 // Screen reader only content
-.offscreen {
-  position: absolute;
-  left: -10000px;
-  top: auto;
-  width: 1px;
-  height: 1px;
-  overflow: hidden;
-
-  // Screen reader toggleable function, including offscreen navigation
-  &:active,
-  &:focus {
-    left: 5px;
-    top: 5px;
-    width: auto;
-    height: auto;
-    overflow: auto;
-  }
+// code taken from orange's fork of bootstrap
+// https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/v5.1.3/scss/mixins/_visually-hidden.scss#L8
+.offscreen,
+.offscreen-focusable:not(:focus) {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  padding: 0 !important;
+  margin: -1px !important;
+  overflow: hidden !important;
+  clip: rect(0, 0, 0, 0) !important;
+  white-space: nowrap !important;
+  border: 0 !important;
```

Comment:
https://css-tricks.com/comparing-various-ways-to-hide-things-in-css/#aa-method-5-the-visually-hidden-class

From this article, I think you should add `clip-rect` below `clip` as this last one has been deprecated in favor of the first.

Furthermore, I think you should remove `!important`

Reply:
I'm with @jbpenrath on the `!important` part. It makes things more complex to debug if we ever override something in the future.

Reply:
You meant `clip-path` I guess. There were a few discussions on this in the bootstrap project, and not using it is intentional, even when knowing it's deprecated, i'll let you check https://github.com/twbs/bootstrap/pull/25197

Reply:
Will remove the `!important` bits ;)

Reply:
So what do you think @jbpenrath about the clip-path issue? Seems to have noticeable perfs issues because of a Chrome bug. My choice would be to leave it as is, but as you wish, let me know.

Reply:
I didn't know about that. Since february 2018, the problem seems to be fixed. I tried an example from my chromium based browser and there is no more painting issue. https://bugs.chromium.org/p/chromium/issues/detail?id=611257